tools/extras/install_irstlm.sh

```diff
@@ -55,6 +55,7 @@ fi
   [ ! -z ${IRSTLM} ] && \
     echo >&2 "IRSTLM config is already in env.sh" && exit
 
+  wd=`pwd`
```

Comment:
why is this line added, isn't it overridden by the next line?
the '|| pwd' is about BSD compatibility.